rest/data/interface.go

```diff
@@ -42,27 +42,32 @@ type Connector interface {
 	// Buildlogger Log
 	//////////////////
 	// FindLogByID returns the buildlogger log with the given id as a
-	// LogIterator with the corresponding time range.
-	FindLogByID(context.Context, string, util.TimeRange) (io.Reader, error)
+	// LogIterator with the corresponding time range. If the bool is set to
+	// true, the log lines should be concatenated with their timestamps.
+	FindLogByID(context.Context, string, util.TimeRange, bool) (io.Reader, error)
```

Comment:
I'm not a huge fan of printTime as a boolean argument, but aside from putting these all in as arg structs I don't really have a better idea?